### PR TITLE
Add minimal image upload example

### PR DESCRIPTION
**Pull request by trishalim**
todo

- [ ] deploy demo

### CODE REVIEW

**Review by aeplay — APPROVED** [review]

File: examples/clerk/README.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
-This is a minimal example of how to use Clerk authentication with Jazz.
+Live version: https://clerk-demo.jazz.tools/
+
+## Installing & running the example locally
+
+(this requires `pnpm` to be installed, see [https://pnpm.io/installation](https://pnpm.io/installation))
+
+Start by checking out `jazz`
+
+```bash
```
Comment (bensleveritt) [inline examples/clerk/README.md]: We should simplify these instructions. If you're having to reach for `pnpm pack` during an install, that's probably too many steps for someone just wanting to try something out. I'd suggest: 

- `npx degit gardencmp/jazz jazz`
- `cd jazz/examples/chat`
- `pnpm i`
- `npx turbo build`
- `pnpm dev`

Comment (bensleveritt) [reply examples/clerk/README.md]: In fact, the current instructions miss the repo build step, so I don't think it'll work. You need to run `pnpm install` before `pnpm pack`

Comment (trishalim) [reply examples/clerk/README.md]: That's much better and it works for me! Thanks

Comment (trishalim) [reply examples/clerk/README.md]: I will do it in a separate PR and apply to all the examples

**Pull request merged**